civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -2422,6 +2422,12 @@ function civihr_employee_portal_get_vacancy_list() {
         'return' => "id,start_date,end_date,status_id,description,position,salary,location",
     ));
 
+  $result_output['values'] = civihr_employee_portal_convert_array_option_value(
+    'hrjc_location',
+    'location',
+    $result_output['values']
+  );
+
     // Output json
     drupal_json_output($result_output);
     drupal_exit();
```

Comment:
Can you please indent the rest of the function? It will look weird to have only the new code indented with two spaces